assets/app/views/createProject.html

```diff
@@ -0,0 +1,66 @@
+<div ng-controller="CreateProjectController">
+  <div class="container">
+    <div class="col-md-12">
+      <h1>Create Project</h1>
+      <form name="createProjectForm">
+        <div class="form-group">
+          <label for="name">Name *</label>
+          <span ng-class="{'has-error': createProjectForm.name.$error.pattern || nameTaken}">
+            <input class="form-control input-lg"
+                name="name"
+                id="name"
+                placeholder="my-project"
+                type="text" autofocus
+                required
+                maxlength="63"
```

Comment:
go ahead and add minlength="2" (c.f. https://github.com/openshift/origin/issues/1232#issuecomment-100232543)